bitcoin/tests/serde.rs

```diff
@@ -109,10 +109,9 @@ fn serde_regression_txin() {
 
 #[test]
 fn serde_regression_txout() {
-    let txout = TxOut {
-        value: Amount::from_sat(0xDEADBEEFCAFEBABE),
-        script_pubkey: ScriptBuf::from(vec![0u8, 1u8, 2u8]),
-    };
+    let txout =
+        TxOut { value: Amount::MAX_MONEY, script_pubkey: ScriptBuf::from(vec![0u8, 1u8, 2u8]) };
```

Comment:
I think we just stick to `Amount::MAX` in general.  I'd almost be tempted to say `MAX_MONEY` is just a private const.

Reply:
IMO `MAX_MONEY` is more explicit and therefore more clear.